triggers/README.md

```diff
@@ -1,13 +1,25 @@
-# Create a build trigger for a GitHub repository
+# Cloud Build triggers for a GitHub repository
 
 We use Google Cloud Build (GCB) triggers to test the applications.
 
-*   Run the following command to create a trigger for your Kubernetes
+For more information, see the
+[`gcloud alpha builds triggers`](https://cloud.google.com/sdk/gcloud/reference/alpha/builds/triggers/)
+commands.
+
+## Create or update a trigger
+
+*   Run the following command to create or update a trigger for your Kubernetes
     application:
 
+    > **NOTE**: To update the trigger, the configuration file must contain the
+    > trigger ID or the name of the existing trigger, otherwise a new trigger
+    > will be created.
```

Comment:
> **NOTE**: You can only update an existing trigger if its
    > configuration file contains either that trigger's name or trigger
    > ID. If neither are present within the configuration file, a new
    > trigger will be created.